dashboard/config/scripts/alltheplcthings.script

```diff
@@ -31,6 +33,13 @@ level 'U2L3 Lesson Overview', named: true
 stage 'Lesson 2,10', flex_category: 'content'
 level 'U2L10 Lesson Overview', named: true
 
+stage 'Test stage with bubbles'
```

Comment:
Had to move this stage and its levels up since that is what would happen before with flex categories and we no longer want to support writing stages out of order because we don't use it anywhere. This is just the test script for plc things so its the only place it came up.

Reply:
Thanks for explaining! I was wondering.